### PR TITLE
XBGE Blog

### PR DESCRIPTION
**Pull request by nmauersberg**

### CODE REVIEW

**Review by Valioesi — APPROVED** [review]
We still need to add the logic to fetch the blog from directus.expedition also on the volksentscheid page. Maybe I can do that the next days. Otherwise next week. 

Design looks good. Thanks!

File: components/Blog/BlogPost/index.tsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  title: string;
+  content: string;
+  assetId: string;
+  date: string;
+};
+
+export const PostFull = ({ title, content, assetId, date }: BlogProps) => {
+  return (
+    <>
+      {/* <Helmet>
```
[the remark this answers is not in the document]
Comment (nmauersberg) [reply components/Blog/BlogPost/index.tsx:18]: It's just commented out until its working again. I just wanted to keep the old structure. :)

File: components/Section/index.tsx [hunk 2]
```diff
@@ -255,8 +255,8 @@ export const Section = ({ section }: SectionProps): ReactElement => {
                       );
                     case 'sectionsComponent':
                       const Component = dynamic(
-                        () => import(`../_dynamic/${element.component}`)
-                        // { ssr: false, loading: () => null }
+                        () => import(`../_dynamic/${element.component}`),
+                        { ssr: false, loading: () => null }
```
[the remark this answers is not in the document]
Comment (nmauersberg) [reply components/Section/index.tsx:266]: Yes. It leads to runtime errors sometimes. I'd suggest that there might be a bug in the next/dynamic loader and I am nut sure, if it is only a dev server issue, but I had an error sometimes, navigating from subpages back to home. It happens rarely though.

[the remark this answers is not in the document]
Comment (Valioesi) [reply components/Section/index.tsx:266]: Weird.

**Pull request merged**